contracts/deposit.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.11;
+import "hardhat/console.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {ILendingPool} from "@aave/protocol-v2/contracts/interfaces/ILendingPool.sol";
+import {ILendingPoolAddressesProvider} from "@aave/protocol-v2/contracts/interfaces/ILendingPoolAddressesProvider.sol";
+
+contract AaveMiddleware{
+
+    function getLendingPoolAddress() internal returns(address){
+        string  _LendingPoolAddressProviderAddr="0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5"; //Mainnet
```

Comment:
Solidity has an `address` datatype, you can use it.
```suggestion
        address  _LendingPoolAddressProviderAddr=0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5; //Mainnet
```

Reply:
Corrected